### ISSUE TITLE
addition of filter_expr in log_config to be able to set true and false

### ISSUE THREAD

**Issue by ramimmadi** [terraform-google-modules/terraform-google-network#353]
### TL;DR

addition of filter_expr in log_config to be able to set true and false

### Terraform Resources

```markdown
google_compute_subnetwork
```

### Detailed design

```markdown
filter_expr
```

### Additional information

[filter_expr](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_subnetwork#filter_expr) - (Optional) Export filter used to define which VPC flow logs should be logged, as as CEL expression. See https://cloud.google.com/vpc/docs/flow-logs#filtering for details on how to format this field. The default value is 'true', which evaluates to include everything.

https://github.com/terraform-google-modules/terraform-google-network/blob/v5.0.0/modules/subnets/main.tf#L40

**Comment by bharathkkb** [comment]
@ramimmadi Thanks for the FR! Happy to review a PR adding this

**Comment by bharathkkb** [comment]
fixed by https://github.com/terraform-google-modules/terraform-google-network/pull/360
